aurorastation.dme

```diff
@@ -1250,7 +1250,6 @@
 #include "code\modules\clothing\ears\skrell.dm"
 #include "code\modules\clothing\glasses\glasses.dm"
 #include "code\modules\clothing\glasses\hud.dm"
-#include "code\modules\clothing\gloves\arm_guard.dm"
```

Comment:
Why is this being removed?

Reply:
This code is from a previous PR, please put it back.

Reply:
I'm sorry